components/properties/Cargo.toml

```diff
@@ -52,3 +54,4 @@ std = ["icu_provider/std"]
 default = []
 serde = ["dep:serde", "zerovec/serde", "icu_uniset/serde", "icu_codepointtrie/serde"]
 datagen = ["serde", "zerovec/serde_serialize", "icu_uniset/serde_serialize", "icu_codepointtrie/serde_serialize"]
+unicode-bidi = [ "dep:unicode-bidi" ]
```

Comment:
Suggestion: name the feature just `"bidi"`

Reply:
done